src/renderer/index.ts

```diff
@@ -44,11 +45,35 @@ async function setupVue() {
 
 async function setupI18n() {
     const myLocale = Intl.DateTimeFormat().resolvedOptions().locale.split("-")[0];
-    const messages: Record<string, Record<string, string>> = {};
+    // `any` required as these are loaded from json files with no defined format
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const messages: Record<string, Record<string, any>> = {};
+    const translationFiles: Record<string, Array<string>> = {};
+
     for (const filePath in localeFilePaths) {
-        const localeCode = filePath.match(/([a-z]{2})\.json$/)![1];
-        messages[localeCode] = localeFilePaths[filePath];
+        const localeCode = filePath.match(/\/([a-z]{2})\/.+?\.json$/)![1];
+        const translationFileURI = `${localeFilePaths[filePath]}`;
```

Comment:
Just `localeFilePaths[filePath]`, no need for string interpolation, this is why the previous bad typing of `localeFilePaths` wasn't cought.